### PR TITLE
Resolve conflicts with `master`

### PR DESCRIPTION
**Pull request by MaxDesiatov**

### CODE REVIEW

File: utils/build-presets.ini [hunk 1]
```diff
@@ -2515,6 +2515,7 @@ swift-install-components=autolink-driver;compiler;clang-builtin-headers;stdlib;s
 llvm-install-components=clang
 install-swift
 install-prefix=/%(TOOLCHAIN_NAME)s/usr
+swift-darwin-supported-archs==x86_64
```
Comment (MaxDesiatov) [inline utils/build-presets.ini:2518]: shouldn't this be a single `=`?

Comment (kateinoigakukun) [reply utils/build-presets.ini]: Right 😅

**Comment by MaxDesiatov** [comment]
@kateinoigakukun tests currently fail with `Caught exception of type "IncorrectCallIndirectSignature"`, but these test binaries do pass validation with `wasm-validate`, so this doesn't seem to be a caller/callee arguments mismatch:

```
******************** TEST 'Swift(wasi-wasm32) :: stdlib/ReflectionHashing.swift' FAILED ********************
Script:
--
: 'RUN: at line 1';   /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/bin/swiftc -target wasm32-unknown-wasi -Xcc --sysroot=/Users/maxd/Documents/swift-source/wasi-sdk/share/wasi-sysroot -Xclang-linker --sysroot=/Users/maxd/Documents/swift-source/wasi-sdk/share/wasi-sysroot -tools-directory /Users/maxd/Documents/swift-source/wasi-sdk/bin -toolchain-stdlib-rpath  -module-cache-path '/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/swift-test-results/wasm32-unknown-wasi/clang-module-cache' -swift-version 4  -Xfrontend -ignore-module-source-info  -module-name a /Users/maxd/Documents/swift-source/swift/test/stdlib/ReflectionHashing.swift -o /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/test-wasi-wasm32/stdlib/Output/ReflectionHashing.swift.tmp.out
: 'RUN: at line 2';   echo /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/test-wasi-wasm32/stdlib/Output/ReflectionHashing.swift.tmp.out
: 'RUN: at line 3';   /usr/bin/env DYLD_LIBRARY_PATH='/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/lib/swift/wasi/wasm32' LD_LIBRARY_PATH='/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/lib/swift/wasi/wasm32:' SIMCTL_CHILD_DYLD_LIBRARY_PATH='/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/lib/swift/wasi/wasm32' /usr/local/bin/wasmer run --backend cranelift -- /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/test-wasi-wasm32/stdlib/Output/ReflectionHashing.swift.tmp.out
--
Exit Code: 1

Command Output (stdout):
--
/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/test-wasi-wasm32/stdlib/Output
/ReflectionHashing.swift.tmp.out

--
Command Output (stderr):
--
/Users/maxd/Documents/swift-source/swift/test/stdlib/ReflectionHashing.swift:21:7: warning: variable 'expected' was never mutated; consider changing to 'let' constant
  var expected = "- 0 key/value pairs\n"
  ~~~ ^
  let
Error: error: Caught exception of type "IncorrectCallIndirectSignature".
```

`wasm-validate` passes when I run it on the binary as `wasm-validate build/Ninja-ReleaseAssert/swift-macosx-x86_64/test-wasi-wasm32/stdlib/Output/ReflectionHashing.swift.tmp.out`.

This is reproducible on CI and you can also see these errors in the logs.

Do you have any suggestions?

**Comment by kateinoigakukun** [comment]
@MaxDesiatov Thanks for big efforts. Could you give me the wasm binary?

**Comment by MaxDesiatov** [comment]
I've got it reproduced with `wasminspect` too:

```
Function exec failure IndirectCallTypeMismatch("$s14StdlibUnittest14_allTestSuites_WZ", FuncType { form: Func, params: [I32], returns: [] }, FuncType { form: Func, params: [], returns: [] })
```

Now I think I'll be able to fix it myself, but here's the binary just in case

[ReflectionHashing.swift.tmp.out.zip](https://github.com/swiftwasm/swift/files/5170382/ReflectionHashing.swift.tmp.out.zip)

**Comment by MaxDesiatov** [comment]
The stack trace is:

```
0: swift_once
1: StdlibUnittest.TestSuite.__allocating_init(Swift.String) -> StdlibUnittest.TestSuite
2: main
3: __main_argc_argv
4: __main_void
5: __original_main
6: _start
```

And the current definition of `swift_once` is

```c++
void swift::swift_once(swift_once_t *predicate, void (*fn)(void *),
                       void *context) {
#ifdef SWIFT_STDLIB_SINGLE_THREADED_RUNTIME
  if (! *predicate) {
    *predicate = true;
    fn(context);
  }
#elif defined(__APPLE__)
  dispatch_once_f(predicate, context, fn);
#elif defined(__CYGWIN__)
  _swift_once_f(predicate, context, fn);
#else
  std::call_once(*predicate, [fn, context]() { fn(context); });
#endif
}
```

We're using `SWIFT_STDLIB_SINGLE_THREADED_RUNTIME` branch now, and my current hypothesis is that initialization of global variables puts them into a closure invoked with `swift_once`, but that closure doesn't expect any context, and `swift_once` attempts to pass it with `fn(context)`. I don't have much knowledge where exactly this globals initialization code is emitted, lmk if you figured it out.

**Comment by MaxDesiatov** [comment]
I've resolved the previously mentioned issues, but now the build fails with these errors:

```
FAILED: stdlib/public/core/WASI/wasm32/Swift.o 
cd /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/stdlib/public/core && /Applications/Xcode-12.0.0-beta.6.app/Contents/Developer/usr/bin/python3 /Users/maxd/Documents/swift-source/swift/utils/line-directive @/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/stdlib/public/core/64396f5c7aa443563da6adb5851e1ec99f3cbf4c.txt -- /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./bin/swiftc -c -sdk /Users/maxd/Documents/swift-source/wasi-sdk/share/wasi-sysroot -target wasm32-unknown-wasi -Xcc -D_WASI_EMULATED_MMAN -resource-dir /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift -O -D INTERNAL_CHECKS_ENABLED -D SWIFT_ENABLE_RUNTIME_FUNCTION_COUNTERS -D SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING -module-cache-path /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./module-cache -no-link-objc-runtime -enable-library-evolution -Xfrontend -assume-single-threaded -Xfrontend -enforce-exclusivity=unchecked -nostdimport -parse-stdlib -module-name Swift -Xfrontend -group-info-path -Xfrontend /Users/maxd/Documents/swift-source/swift/stdlib/public/core/GroupInfo.json -swift-version 5 -runtime-compatibility-version none -disable-autolinking-runtime-compatibility-dynamic-replacements -warn-swift3-objc-inference-complete -Xfrontend -verify-syntax-tree -Xllvm -sil-inline-generics -Xllvm -sil-partial-specialization -Xfrontend -enable-experimental-concise-pound-file -Xcc -DswiftCore_EXPORTS -warn-implicit-overrides -module-link-name swiftCore -whole-module-optimization -parse-as-library -I /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/wasi/wasm32 -o /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/stdlib/public/core/WASI/wasm32/Swift.o @/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/stdlib/public/core/64396f5c7aa443563da6adb5851e1ec99f3cbf4c.txt
<unknown>:0: error: file '/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/
module-cache/M30B55G98BUW/SwiftShims-3RU46TWTDSB76.pcm'
is not a valid precompiled module file
/Users/maxd/Documents/swift-source/swift/stdlib/public/core/ArrayBody.swift:18:8: 
error: no such module 'SwiftShims'
import SwiftShims
       ^
<module-includes>:4:10: note: in file included from <module-includes>:4:
#include "GlobalObjects.h"
         ^
/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/shims/GlobalObjects.h:23:10: note: in file included from /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/shims/GlobalObjects.h:23:
#include "HeapObject.h"
         ^
/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/shims/HeapObject.h:16:10: note: in file included from /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/shims/HeapObject.h:16:
#include "SwiftStddef.h"
         ^
/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/shims/SwiftStddef.h:25:10: note: in file included from /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/shims/SwiftStddef.h:25:
#include <stddef.h>
         ^
[1044/1136][ 91%][171.840s] cd /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/stdlib/public/core && /usr/local/Cellar/cmake/3.18.2/bin/cmake -E remove -f /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/wasi/wasm32/Swift.swiftmodule /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/wasi/wasm32/Swift.swiftdoc /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/wasi/wasm32/Swift.swiftinterface && /usr/local/Cellar/cmake/3.18.2/bin/cmake -E make_directory /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/wasi/wasm32 && /Applications/Xcode-12.0.0-beta.6.app/Contents/Developer/usr/bin/python3 /Users/maxd/Documents/swift-source/swift/utils/line-directive @/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/stdlib/public/core/64396f5c7aa443563da6adb5851e1ec99f3cbf4c.txt -- /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./bin/swiftc -emit-module -o /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/wasi/wasm32/Swift.swiftmodule -avoid-emit-module-source-info -sdk /Users/maxd/Documents/swift-source/wasi-sdk/share/wasi-sysroot -target wasm32-unknown-wasi -Xcc -D_WASI_EMULATED_MMAN -resource-dir /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift -O -D INTERNAL_CHECKS_ENABLED -D SWIFT_ENABLE_RUNTIME_FUNCTION_COUNTERS -D SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING -module-cache-path /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./module-cache -no-link-objc-runtime -enable-library-evolution -Xfrontend -assume-single-threaded -Xfrontend -enforce-exclusivity=unchecked -nostdimport -parse-stdlib -module-name Swift -Xfrontend -group-info-path -Xfrontend /Users/maxd/Documents/swift-source/swift/stdlib/public/core/GroupInfo.json -swift-version 5 -runtime-compatibility-version none -disable-autolinking-runtime-compatibility-dynamic-replacements -warn-swift3-objc-inference-complete -Xfrontend -verify-syntax-tree -Xllvm -sil-inline-generics -Xllvm -sil-partial-specialization -Xfrontend -enable-experimental-concise-pound-file -Xcc -DswiftCore_EXPORTS -warn-implicit-overrides -module-link-name swiftCore -whole-module-optimization -parse-as-library -I /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/wasi/wasm32 -emit-module-interface-path /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/./lib/swift/wasi/wasm32/Swift.swiftinterface @/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/stdlib/public/core/64396f5c7aa443563da6adb5851e1ec99f3cbf4c.txt
ninja: build stopped: subcommand failed.
ERROR: command terminated with a non-zero exit status 1, aborting
```

I've checked the file type, but I'm not sure what's the best way to check if it's a walid precompiled module file:

```
% file /Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/\
module-cache/M30B55G98BUW/SwiftShims-3RU46TWTDSB76.pcm
/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/swift-macosx-x86_64/
module-cache/M30B55G98BUW/SwiftShims-3RU46TWTDSB76.pcm: 
WebAssembly (wasm) binary module version 0x1 (MVP)

**Comment by kateinoigakukun** [comment]
@MaxDesiatov 

After this merge https://github.com/swiftwasm/swift-corelibs-foundation/pull/121, building Foundation failed with the following errors. Could you take look at this?

```
2020-09-14T03:39:27.9284950Z /Users/runner/work/swift/swift-corelibs-foundation/Sources/Foundation/Data.swift:86:20: error: global function 'calloc' cannot be used in an '@inlinable' function because 'CoreFoundation' was imported implementation-only
2020-09-14T03:39:27.9285930Z             return calloc(1, size)
2020-09-14T03:39:27.9286170Z                    ^
2020-09-14T03:39:27.9287590Z /Users/runner/work/swift/swift-corelibs-foundation/Sources/Foundation/Data.swift:88:20: error: global function 'malloc' cannot be used in an '@inlinable' function because 'CoreFoundation' was imported implementation-only
2020-09-14T03:39:27.9288570Z             return malloc(size)
2020-09-14T03:39:27.9288800Z                    ^
2020-09-14T03:39:27.9290180Z /Users/runner/work/swift/swift-corelibs-foundation/Sources/Foundation/Data.swift:272:17: error: global function 'free' cannot be used in an '@inlinable' function because 'CoreFoundation' was imported implementation-only
2020-09-14T03:39:27.9291380Z                 free(bytes)
2020-09-14T03:39:27.9291590Z                 ^
2020-09-14T03:39:27.9293020Z /Users/runner/work/swift/swift-corelibs-foundation/Sources/Foundation/Data.swift:291:13: error: global function 'memset' cannot be used in an '@inlinable' function because 'CoreFoundation' was imported implementation-only
2020-09-14T03:39:27.9294460Z             memset(_bytes! + origLength, 0, newLength - origLength)
2020-09-14T03:39:27.9294810Z             ^
2020-09-14T03:39:27.9296200Z /Users/runner/work/swift/swift-corelibs-foundation/Sources/Foundation/Data.swift:806:15: error: global function 'memset' cannot be used in an '@inlinable' function because 'CoreFoundation' was imported implementation-only
2020-09-14T03:39:27.9297930Z               memset(rawBuffer.baseAddress!.advanced(by: range.lowerBound), 0, range.upperBound - range.lowerBound)
2020-09-14T03:39:27.9298540Z               ^
2020-09-14T03:39:27.9299930Z /Users/runner/work/swift/swift-corelibs-foundation/Sources/Foundation/Data.swift:1966:17: error: global function 'memset' cannot be used in an '@inlinable' function because 'CoreFoundation' was imported implementation-only
2020-09-14T03:39:27.9301090Z                 memset(buffer.baseAddress!, Int32(repeatedValue), buffer.count)
2020-09-14T03:39:27.9301500Z                 ^
2020-09-14T03:39:27.9302880Z /Users/runner/work/swift/swift-corelibs-foundation/Sources/Foundation/Data.swift:2661:21: error: global function 'memcpy' cannot be used in an '@inlinable' function because 'CoreFoundation' was imported implementation-only
2020-09-14T03:39:27.9304180Z                 _ = memcpy(UnsafeMutableRawPointer(buffer.baseAddress!), bytes.baseAddress!, cnt)
2020-09-14T03:39:27.9304910Z                     ^
2020-09-14T03:39:27.9306380Z /Users/runner/work/swift/swift-corelibs-foundation/Sources/Foundation/Data.swift:2757:28: error: global function 'memcmp' cannot be used in an '@inlinable' function because 'CoreFoundation' was imported implementation-only
2020-09-14T03:39:27.9307490Z                     return memcmp(b1.baseAddress!, b2.baseAddress!, b2.count) == 0
2020-09-14T03:39:27.9307850Z                            ^
```

**Comment by MaxDesiatov** [comment]
I'm not sure how to fix this and keep `@_implementationOnly` present at the same time. Although after I removed that attribute, compilation no longer fails in `Data.swift` but it fails in the end of the build with `<unknown>:0: error: unable to execute command: <unknown>`.

When I try to run the build command with `-v`, I see that the last command executed by the driver is `swift-autolink-extract`, but when I run `swift-autolink-extract` separately with the exact same arguments it succeeds. It looks like `unable to execute command` is printed by the `swiftc` driver, but I'm still not sure what exactly causes it.

**Comment by MaxDesiatov** [comment]
For posterity, here's the build command that fails:

```
/Users/maxd/Documents/swift-source/host-toolchain-sdk/swift-wasm-DEVELOPMENT-SNAPSHOT-2020-09-14-a/usr/bin/swiftc -incremental -v -output-file-map Sources/Foundation/CMakeFiles/Foundation.dir//output-file-map.json -emit-library -static -o lib/libFoundation.a -module-name Foundation -module-link-name Foundation -emit-module -emit-module-path swift/Foundation.swiftmodule -emit-dependencies -DDEPLOYMENT_RUNTIME_SWIFT -Xcc -F/Users/maxd/Documents/swift-source/build/Ninja-ReleaseAssert/foundation-wasi-wasm32 -sdk /Users/maxd/Documents/swift-source/wasi-sdk/share/wasi-sysroot -target wasm32-unknown-wasi -I CoreFoundation.framework/Headers /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/AffineTransform.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Array.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Boxing.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Bridging.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Bundle.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/ByteCountFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Calendar.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/CGFloat.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/CharacterSet.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Codable.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Collections+DataProtocol.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/ContiguousBytes.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Data.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/DataProtocol.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Date.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/DateComponents.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/DateComponentsFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/DateFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/DateInterval.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/DateIntervalFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Decimal.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Dictionary.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/DispatchData+DataProtocol.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/EnergyFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/ExtraStringAPIs.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/FileHandle.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/FileManager.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Formatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/FoundationErrors.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/IndexPath.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/IndexSet.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/ISO8601DateFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/JSONEncoder.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/JSONSerialization.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/LengthFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Locale.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/MassFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Measurement.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/MeasurementFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSArray.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSAttributedString.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCache.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCalendar.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCFArray.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCFBoolean.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCFCharacterSet.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCFDictionary.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCFSet.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCFString.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCFTypeShims.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCharacterSet.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCoder.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSComparisonPredicate.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSCompoundPredicate.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSConcreteValue.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSData+DataProtocol.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSData.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSDate.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSDecimalNumber.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSDictionary.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSEnumerator.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSError.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSExpression.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSGeometry.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSIndexPath.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSIndexSet.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSLocale.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSLock.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSLog.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSMeasurement.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSNull.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSNumber.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSObjCRuntime.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSObject.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSOrderedSet.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSPathUtilities.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSPersonNameComponents.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSPlatform.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSPredicate.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSRange.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSRegularExpression.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSSet.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSSortDescriptor.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSString.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSStringAPI.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSSwiftRuntime.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSTextCheckingResult.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSTimeZone.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSURL.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSURLError.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSUUID.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NSValue.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/NumberFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Operation.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/PersonNameComponents.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/PersonNameComponentsFormatter.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Pointers+DataProtocol.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Progress.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/ProgressFraction.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/PropertyListSerialization.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/ReferenceConvertible.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Scanner.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/ScannerAPI.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Set.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/String.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/StringEncodings.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/TimeZone.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/Unit.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/URL.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/URLComponents.swift /Users/maxd/Documents/swift-source/swift-corelibs-foundation/Sources/Foundation/UUID.swift
```

**Comment by MaxDesiatov** [comment]
It's green! 🥳

**Comment by MaxDesiatov** [comment]
Looks like tests are failing on Linux: `wasm-ld: warning: unexpected existing value for R_WASM_MEMORY_ADDR_SLEB: existing=126120 expected=126412`

**Comment by MaxDesiatov** [comment]
This error is not available in the logs unfortunately, was the Linux build killed because of the 5 hour limit? 🤔

**Comment by kateinoigakukun** [comment]
Finally, CI passed

**Review by kateinoigakukun — APPROVED** [review]

**Pull request merged**